lib/plurimath/formatter/number_formatter.rb

```diff
@@ -0,0 +1,115 @@
+require "twitter_cldr"
+require_relative "twitter_cldr_rb/integer"
+require_relative "twitter_cldr_rb/fraction"
+require_relative "twitter_cldr_rb/number_formatter"
+
+module Plurimath
+  module Formatter
+    class NumberFormatter
+      attr_accessor :locale, :localize_number, :localiser_symbols
+
+      LOCALIZE_NUMBER_REGEX = %r{(?<group>[^#])?(?<groupdigits>#+0)(?<decimal>.)(?<fractdigits>#+)(?<fractgroup>[^#])?}
+      SUPPORTED_NOTATIONS = %i[e scientific engineering].freeze
+
+      def initialize(locale, localize_number:, localiser_symbols:)
```

Comment:
I think we should be using `s` or `z` in localize everywhere in plurimath.
 Having `localize_number` along with `localiser_symbol` does not feels right to me.